.github/workflows/build_and_test.yml

```diff
@@ -306,6 +306,12 @@ jobs:
         name: unit-tests-log-${{ matrix.modules }}-${{ matrix.comment }}-${{ matrix.java }}-${{ matrix.hadoop }}-${{ matrix.hive }}
         path: "**/target/unit-tests.log"
 
+  maven-test:
```

Comment:
To ensure the fix, let's run a maven test. Once successful, this will be revert

Reply:
maven tests passed : https://github.com/LuciferYang/spark/runs/29157086061

![image](https://github.com/user-attachments/assets/b5cbef70-895c-4de2-8ea2-bb1aed1bf0c6)


reverted